lib/rules/static-property-placement.js

```diff
@@ -129,9 +129,17 @@ module.exports = {
       ClassProperty: node => reportNodeIncorrectlyPositioned(node, STATIC_PUBLIC_FIELD),
 
       MemberExpression: node => {
+        // If definition type is undefined then it must not be a defining expression or if the definition is inside a
+        // class body then skip this node.
+        const { right } = node.parent;
```

Comment:
whoops, looks like this breaks in node 4? if so:
```suggestion
        const right = node.parent.right;
```